scripts/check-system-of-record.sh

```
@@ -0,0 +1,93 @@
+# docstrings/comments that mention the method names. The runtime
+# regression coverage lives in the unit + E2E tests.
+violations=$(
+  for dir in "${SCOPE_DIRS[@]}"; do
+    [ -d "$dir" ] || continue
+    grep -rEn --include='*.ts' --include='*.tsx' --include='*.js' --include='*.sh' \
+      "$COMBINED" "$dir" 2>/dev/null || true
+  done \
+    | grep -vE 'gbrain-allow-direct-insert:' \
+    | grep -vE ':[[:space:]]*\*[[:space:]]+' \
+    | grep -vE ':[[:space:]]*//' \
+    | grep -vE '`[^`]*\\.[[:alnum:]_]+\(' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)